CHANGELOG.rst

```diff
@@ -2,6 +2,58 @@
 Changelog for package rviz_visual_tools
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+3.9.0 (2020-06-04)
+------------------
+* replace tf_conversions with tf2 (`#151 <https://github.com/PickNikRobotics/rviz_visual_tools/issues/151>`_)
+  Co-authored-by: Bjar Ne <gleichdick@users.noreply.github.com>
```

Comment:
Should the Co-authors be mentioned in-line? I thought i's sufficient to list them at the bottom.